<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>NumberPhone</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Date of delivery</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="18" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name of Product</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Number of Passport</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Patronymic</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Driver</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ExtraInfo</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Provider of product</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="typeEdit"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="countEdit"/>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>At Storage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QComboBox" name="drivercomboBox"/>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QLineEdit" name="extrainfoEdit"/>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLineEdit" name="famEdit"/>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QLineEdit" name="nametEdit"/>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QLineEdit" name="ochEdit"/>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QLineEdit" name="numphEdit"/>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="QLineEdit" name="companyEdit"/>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="QLineEdit" name="numpsEdit"/>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
